### DISCUSSION TITLE
Bookmarks

### DISCUSSION THREAD

**Discussion by Silther** [libre-tube/LibreTube#2123] [Q&A] [answered]
What exactly are bookmarks and what do you use it for?

**Comment by Bnyro** [comment, marked as answer]
Currently there are only bookmarks for playlists, which means that you find them easier, might be useful for music playlists that get constantly updated

**Reply by silentTeee** [reply]
So I have a naïve follow-up to this question: how does one _apply_ a bookmark? I've been looking around for that and the only thing I seem to be able to find is the option to delete all bookmarks, but not to add any. Could it be that the UI elements for it were accidentally removed at some point...?

**Reply by Bnyro** [reply]
No, it's not. There is a bookmark button when opening a YT playlist.

**Reply by silentTeee** [reply]
..._OH,_ as in a playlist that doesn't belong to you, for future reference? I don't know why, but I got it in my head that I should be able to apply one to _my own_ playlists, and I was wondering why I couldn't spot a bookmark button for it.

**Reply by Bnyro** [reply]
Yes, not your own playlists.

**Reply by silentTeee** [reply]
Okay, that makes much more sense. Thanks for clarifying, clearly I had not gotten enough sleep when I asked that question. 

Enjoy your day, and thank you for all the careful consideration you have put into this app! It's amazing how smoothly it runs, I hope to one day be able to write apps as nice as this one.
